flatdata-cpp/test/BackwardCompatibilityTest.cpp

```diff
@@ -137,12 +137,13 @@ std::array< uint8_t, 66 > expected_multivector_data
        "\x00\xff\xff\xff\xff\xef\xbe\xad\xde"          // Payload
        "\x00\x00\x00\x00\x00\x00\x00\x00"};            // Padding
 
-std::array< uint8_t, 37 > expected_multivector_index
-    = {"\x14\x00\x00\x00\x00\x00\x00\x00"    // Index size in bytes
+std::array< uint8_t, 42 > expected_multivector_index
```

Comment:
As nobody uses us yet (except us), this might be okay, but this comment in README.md should then transform into:

> ....we are changing APIs and binary layout so please expect that....

This will also render all previously built archives inoperable.
The earlier accepted way to do it was to do `mutlivector2` in schema. Lets talk.

Reply:
Not necessary, binary layout is the same (test was just creating broken structures without sentinel)

Reply:
oki